priv/sites/zotonic_status/modules/mod_zotonic_site_management/support/zotonic_status_addsite.erl

```diff
@@ -42,6 +43,8 @@ addsite(Name, Options, Context) when is_binary(Name) ->
     end.
 
 % Check Hostname (must have DNS resolve)
+-spec addsite_check_hostname(binary(), list(), #context{}) ->
```

Comment:
The spec in line 46 uses a record, please define a type for the record and use that instead.